src/NLog.proj

```diff
@@ -1,578 +0,0 @@
-<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
We need this file for generation the XSD afaik

Reply:
I have removed NLog.proj. Not sure I understand the comment about XSD.

Reply:
I'm guessing the XSD is the Intellisense for NLog configuration files. Think it should be a different PR that extracts the XSD logic from the proj-file. Will "undelete" the file, so it is not lost.